### PR TITLE
Jenkinsfile: Build PR only on nRF platform

### PR DESCRIPTION
**Pull request by KAGA164**
PR should be only build on nRF platform to
save a time. Master and other branch are tested
in default configuration.

Signed-off-by: Kamil Gawor <Kamil.Gawor@nordicsemi.no>

### CODE REVIEW

**Comment by KAGA164** [comment]
@thst-nordic Please review.

File: Jenkinsfile [hunk 1]
```diff
@@ -94,10 +95,23 @@ pipeline {
...
               sh "cat /opt/zephyr-sdk/sdk_version"
-	      sh "source zephyr-env.sh && \
-                  (./scripts/sanitycheck $SANITYCHECK_OPTIONS $ARCH || \
-                  (sleep 10; ./scripts/sanitycheck $SANITYCHECK_OPTIONS $SANITYCHECK_RETRY) || \
-                  (sleep 10; ./scripts/sanitycheck $SANITYCHECK_OPTIONS $SANITYCHECK_RETRY_2))"
+
+              script {
+                // IF PR test only nRF platform.
+	          if (env.CHANGE_TARGET) {
+PLATFORM
```
Comment (mbolivar) [inline Jenkinsfile]: New to Jenkinsfile syntax -- what does this line do?

File: Jenkinsfile [hunk 2]
```diff
@@ -26,6 +26,7 @@ pipeline {
 
       // ENVs for sanitycheck
       ARCH = "-a arm"
+      PLATFORM = "-p nrf9160_pca10090 -p nrf52_pca10040 -p nrf52840_pca10056"
```
Comment (mbolivar) [inline Jenkinsfile]: There are additional boards with upstream support:

```
$ ls -d boards/arm/nrf*pca*
boards/arm/nrf51_pca10028     boards/arm/nrf52811_pca10056  boards/arm/nrf52840_pca10059  boards/arm/nrf52_pca10040  boards/arm/nrf9160_pca10090
boards/arm/nrf52810_pca10040  boards/arm/nrf52840_pca10056  boards/arm/nrf52840_pca10090  boards/arm/nrf52_pca20020
```

Is there some way to use the same glob here?

**Comment by thst-nordic** [comment]
I added these changes and a few more in this PR: https://github.com/NordicPlayground/fw-nrfconnect-zephyr/pull/176

**Comment by thst-nordic** [comment]
PR: #176  ( which includes all of these changes) was merged to master.

**Pull request closed without merging**